### ISSUE TITLE
Question: why do lists require more than one item?

### ISSUE THREAD

**Issue by foldedpaper** [inkle/ink#468]
I am really enjoying working with ink, thanks for offering such a great library. I am using bladecoder's Java implementation of ink to integrate ink scripts into my program, and I have a question about `LIST` variables--why can they not have only one element?

I ask because I sometimes find myself pulling `LIST` variables out of an ink story and using some Java array operations on them. Of course in Java, an array can have only one element, but ink itself won't allow me to create a `LIST` with one element--it points me to creating a `VAR` instead.

I know that ink is very flexible and open to be used in many different ways, so it may not make sense to allow behavior in one area that restricts usage in another. But I'm curious to know why `LIST`s have this restriction. Thanks again.

**Comment by joningold** [comment]
Yeah, it doesn't make any sense. It's mostly because we didn't really think
of implementing it - it's purely on the compiler to figure out what a
single-item list means. You could try making a single "single things list"
list, if you want somewhere to put them!
